electrification.mss

```diff
@@ -221,7 +221,7 @@
        line-color: #00CB66;
     }
 
-    [frequency=null][frequency!=0][voltage>=25000] {
+    [frequency!=null][frequency!=0][voltage>=25000] {
```

Comment:
This always overrules the two selectors after this. Should be `[frequency>0][voltage>=25000]` instead, so that it can be overridden by a subsequent matching selector, while overriding the selector on line 212 when voltage≥25000.